lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -227,7 +227,8 @@ export abstract class PerformanceClient implements IPerformanceClient {
 
         const queueMeasurement: QueueMeasurement = {
             eventName,
-            queueTime,
+            // Always default queue time to 0 for manually completed (improperly instrumented)
+            queueTime: manuallyCompleted ? 0 : queueTime,
```

Comment:
Just a thought, not sure how much I like this idea myself, but what if we make it -1 so we can easily identify bugs versus a legitimate 0?

Reply:
We would then also have to throw out -1s when summing up all the queue times

Reply:
I would prefer -1 than 0.

Reply:
What is the measurement unit: milli seconds?

Reply:
We already have `manuallyCompleted` attribute in place which then get accumulated and tracked. There is no need to set it to `-1`.